enterprise/api/v1/serializers.py

```diff
@@ -97,3 +97,14 @@ class Meta:
     user = UserSerializer()
     enterprise_customer = EnterpriseCustomerSerializer()
     data_sharing_consent = UserDataSharingConsentAuditSerializer(many=True)
+
+
+class EnterpriseCourseEnrollmentSerializer(serializers.ModelSerializer):
+    """
+    Serializer for EnterpriseCourseEnrollment model.
+    """
+    class Meta:
+        model = models.EnterpriseCourseEnrollment
+        fields = (
+            'id', 'enterprise_customer_user', 'course_id', 'consent_granted'
```

Comment:
Thanks @haikuginger! I pushed a commit that exposes `consent_available` and `consent_granted` to the API. I left `consent_granted` in for now, because submitting a new value for `consent_granted` is how you currently update an `EnterpriseCourseEnrollment` object via the API.